Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -82,8 +93,9 @@ void MemoryViewWidget::Update()
   for (int i = 0; i < rows; i++)
   {
     setRowHeight(i, 24);
-
-    u32 addr = m_address - ((rowCount() / 2) * 16) + i * 16;
+    u32 rowmod = ((GetColumnCount(m_type) == 2) ? 4 : 16);
+    u32 addr = m_address - (rowCount() / 2) * rowmod +
+               i * rowmod;  // Alternate view requires two columns with the same address
```

Comment:
Consider placing the comment on top the line rather than on the tail end of it, so it doesn't wrap oddly like this.